### DISCUSSION TITLE
Using ioping tool to test RetroNAS setup for PSX on Mister FPGA.

### DISCUSSION THREAD

**Discussion by toaks2804** [retronas/discussions#8] [Q&A] [answered]
I am wanting to test the speed of the RetroNas server for PSX games. I read over the instructions for using ioping. Do I somehow run it on the Mister itself or do I run it on the Raspberry Pi?

**Comment by sairuk** [comment, marked as answer]
The instructions are running the MiSTer which you can tell by the paths used

https://github.com/danmons/retronas/wiki/MiSTer-FPGA#PSX-core---Latency-performance

you can generally ssh into your MiSTer with user/pass: root/1 and then follow the instructions

**Reply by toaks2804** [reply]
Ok thank you. I confused myself because I didn't know the terminal was accessible on Mister. I appreciate the help.
